p2p/src/authenticated/discovery/actors/tracker/actor.rs

```diff
@@ -294,6 +294,9 @@ mod tests {
     };
     use types::PeerInfo;
 
+    const BITMAP_CHUNK_SIZE: usize = 1;
+    type BitMap = commonware_utils::bitmap::BitMap<BITMAP_CHUNK_SIZE>;
```

Comment:
I removed the `UtilsBitVec` alias since we call it `BitMap` now.

I left the types/variables named `BitVec` in `p2p` as is rather than renaming them to `BitMap` too. (These types are unchanged in this PR.)